flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/cache/FileBasedCache.java

```diff
@@ -97,6 +129,13 @@ public void delete(Path path) {
 
     @Override
     FileCacheEntry internalGet(String key, FileCacheEntry value) {
+        if (metricGroup != null) {
```

Comment:
I am curious why the key parameter is not used here?

Reply:
`Key` is reserved for future use.

`internalGet` is used for some customized operations like statistics.
This PR only focuses on the `hit`, `miss` metric which is not related with `key`.